manifests/h/HERE/OLPCLI/10.5.106/HERE.OLPCLI.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HERE.OLPCLI
+PackageVersion: 10.5.106
+PackageLocale: en-US
+Publisher: HERE Technologies
+PackageName: OLP CLI
+License: Copyright (C) 2017-2022 HERE Global B.V. and its affiliate(s).
```

Comment:
A copyright is not a license. Please move this to the `Copyright` field and provide a valid license